src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+
+        if (user != null && user.getPassword().equals(password)) {
+            return true;
+        }
         return false;
```

Comment:
When we need to return boolean value if/else structure or ternary is not required

instead of
if (condition) {
    return true;
} else {
    return false;
}

we may use just
return condition;